### PR TITLE
Content update

### LINKED ISSUE #409: UAT content edit - confirmation page
Add back in "Keep this number for your records.'

This should be done in English and Spanish.

### PR DESCRIPTION
**Pull request by sharonwarner**
image

image

===

Resolves #409 

- [x] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [x] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -347,7 +347,7 @@
     "alert": "You have successfully submitted your retroactive certification.",
     "alert-returning": "You have successfully submitted your retroactive certification. No further action is required.",
     "header1": "Retroactive Certification Confirmation",
-    "p1": "Your retroactive certification has been submitted. Your confirmation number is <strong>{{confirmationNumber}}</strong>.",
+    "p1": "<p>Your retroactive certification has been submitted. Your confirmation number is <strong>{{confirmationNumber}}</strong>.</p><p>Keep this number for your records.</p>",
```
Comment (kalvinwang) [inline public/locales/en/translation.json:350]: how did this work? officially you can't have second-level tags in i18next-react Trans components, so this <strong>tag</strong> tag shouldn't work, but i see it in your screenshot...

Comment (sharonwarner) [reply public/locales/en/translation.json:350]: Oh interesting! I have no idea 😂 Maybe the bold text won't display in other browsers?

Comment (sharonwarner) [reply public/locales/en/translation.json:350]: I see this in the documentation: `<bold>bold<i>italic</i></b> // no inner elements - only strings!`, but maybe the problem is that `<i>` isn't supported?

Comment (idealisms) [reply public/locales/en/translation.json:350]: Why not just add a new entry to the file for the second paragraph?

Comment (sharonwarner) [reply public/locales/en/translation.json:353]: Oh, I am! It'll be ready in a minute!

**Pull request merged**